media-sound/ario/ario-1.6-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit autotools python-any-r1
```

Comment:
Are autotools or python-any-r1 being used in here at all? Especially if you said python support was removed. If they are not, you should remove this line and the PYTHON_COMPAT above (and test rebuilding the package).